src/shadowbox/README.md

```diff
@@ -1,203 +1,169 @@
-```
+   Add flags for hostname, port, etc. Example:
 
-Use `sudo --preserve-env` if you need to pass environment variables. Use `bash -x` if you need to debug the installation.
+   ```sh
+   sudo bash -c "$(wget -qO- https://raw.githubusercontent.com/Jigsaw-Code/outline-apps/master/server_manager/install_scripts/install_server.sh)" install_server.sh \
+     --hostname=myserver.com \
+     --keys-port=443
+   ```
 
-## Running from source code
+   - Use `sudo --preserve-env` for environment variables.
+   - Use `bash -x` for debugging.
 
-### Prerequisites
+### Running from Source Code
 
-Shadowbox supports running on linux and macOS hosts.
+**Prerequisites**
 
-Besides [Node](https://nodejs.org/en/download/) you will also need:
+- [Docker](https://docs.docker.com/engine/install/)
+- [Node](https://nodejs.org/en/download/) LTS (`lts/hydrogen`, version `18.16.0`)
+- [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) (version `9.5.1`)
 
-1. [Docker Engine](https://docs.docker.com/engine/installation/)
+> [!TIP]
+> If you use `nvm`, switch to the correct Node version with `nvm use`.
 
-### Running Shadowbox as a Node.js app
+1. **Build and Run:**
 
-Build and run the server as a Node.js app:
+   Shadowbox supports running on linux and macOS hosts.
 
-```
-npm run action shadowbox/server/start
-```
+   - **Node.js App**
 
-The output will be at `build/shadowbox/app`.
+     ```sh
+     npm run action shadowbox/server/start
+     ```
 
-### Running Shadowbox as a Docker container
+   - **Docker Container**
 
-### With docker command
+     ```sh
+     npm run action shadowbox/docker/start
+     ```
 
-Build the image and run server:
+     > [!TIP]
+     > Some useful commands when working with Docker images and containers:
+     >
+     > - **Debug Image:**
+     >
+     >   ```sh
+     >   docker run --rm -it --entrypoint=sh localhost/outline/shadowbox
+     >   ```
+     >
+     > - **Debug Running Container:**
+     >
+     >   ```sh
+     >   docker exec -it shadowbox sh
+     >   ```
+     >
+     > - **Cleanup Dangling Images:**
+     >
+     >   ```sh
+     >   docker rmi $(docker images -f dangling=true -q)
+     >   ```
 
-```
-npm run action shadowbox/docker/start
-```
+1. **Send a Test Request**
 
-You should be able to successfully query the management API:
-
-```
-curl --insecure https://[::]:8081/TestApiPrefix/server
-```
-
-To build the image only:
-
-```
-npm run action shadowbox/docker/build
-```
-
-Debug image:
-
-```
-docker run --rm -it --entrypoint=sh localhost/outline/shadowbox
-```
-
-Or a running container:
-
-```
-docker exec -it shadowbox sh
-```
-
-Delete dangling images:
-
-```
-docker rmi $(docker images -f dangling=true -q)
-```
+   ```sh
+   curl --insecure https://[::]:8081/TestApiPrefix/server
+   ```
 
 ## Access Keys Management API
 
-In order to utilize the Management API, you'll need to know the apiUrl for your Outline server.
```

Comment:
The new doc is lacking context. What is this section about? the old text gave some context.

Reply:
I agree that we need some context here, but the old text also gave no context. Which section are you referring to, because I didn't remove anything.

How about something like:

"The Outline Server provides a REST API for access key management. If you know the `apiUrl` of your Outline Server (e.g. `https://1.2.3.4:1234/3pQ4jf6qSr5WVeMO0XOo4z`), you can directly manage the server's access keys using HTTP requests:"

Wdyt?